modules/server/src/network/index.ts

```diff
@@ -35,16 +35,16 @@ const fetchRemoteSchema = async (
 			gqlRequest: { query: getIntrospectionQuery() },
 		});
 
-		if (response.status !== 200) {
-			throw Error('network error');
-		}
-
-		const jsonData = await response.data.json();
-
-		if (jsonData && jsonData.data && isGqlIntrospectionQuery(jsonData.data)) {
-			return { config, introspectionResult: jsonData };
+		if (response.status === 200 && response.statusText === 'OK') {
```

Comment:
Fetch hasn't ever not relied on negatives... Affirmative logic FTW. #accessibility